flake.nix

```diff
@@ -124,6 +124,16 @@
           app = "tests-hifive-inventor";
           toml = "test/tests-hifive-inventor/app.toml";
         };
+        demo-rv64-qemu-virt = hubris {
```

Comment:
Looks like all these targets are run one after the other in nix CI. Is there a way to do these in parallel? Else the CI just keeps getting slower over time.

Reply:
I can set the max-jobs to automatic. see https://github.com/rivosinc/hubris/pull/69

Reply:
Did that make any difference? From the run time, it looks similar or probably worse with this change?

Reply:
Looks like the ubuntu checks went from ~20min to ~10min, but It could just be variation in the github runners.  I am pretty sure in the past I have seen CI runs take 10mins, and some take ~30mins.

Reply:
Ah okay. Probably it can be figured out based on the logs? Anyways, I will mark this comment resolved to move this CL forward, but I think that is something we should check independent of this change.